### ISSUE TITLE
Add twemoji for better emoji rendering across platforms / just for the flags that won't render on windows.

### ISSUE THREAD

**Issue by elinarbur** [benawad/dogehouse#1330]
**Is your feature request related to a problem? Please describe.**
When on windows the emojis for flags on the language selector won't render, using twitter's twemoji library would fix this issue. Of course, flag emojis could just be removed but that also removes the accessibility benefits that they can bring to people with something like dyslexia.

**Describe the solution you'd like**
Using an emoji library like Twitter's twemoji would work perfectly for addressing this issue and could make emoji on Dogehouse look consistent across platforms.

**Describe alternatives you've considered**
Begging Microsoft for flag emojis wouldn't work so the next best thing is just an alternative library for flag emojis.

**Additional context**
Add any other context or screenshots about the feature request here. Screenshot: https://imgur.com/a/xpfc9XF

**Comment by elinarbur** [comment]
This might not be the most important thing in the world but it's a pretty big nice to have.

**Comment by f1shy-dev** [comment]
I'll try work on this.

**Comment by f1shy-dev** [comment]
Native `<select>` doesn't support images, so either we make a popup with a custom dropdown or move this option to settings. I doubt a custom dropdown would be mobile friendly at the bottom of the screen.

**Comment by elinarbur** [comment]
If it's too much of a hassle to do, that's obviously fair enough. Personally, I wouldn't sacrifice mobile-friendliness for a small improvement like this in my own projects.
I still think the consistency benefit would be nice though.

**Comment by f1shy-dev** [comment]
I was thinking to redo the settings area anyway, we could just do it in there.

**Comment by elinarbur** [comment]
Yeah that would probably work perfectly

**Comment by f1shy-dev** [comment]
Imma do that then, would the Footer just show something like
![image](https://user-images.githubusercontent.com/56125930/111873636-36550b00-8989-11eb-995b-1068a6d6ae3e.png)
Where clicking on it takes you to settings?

**Comment by elinarbur** [comment]
Yeah looks good 👍

**Comment by f1shy-dev** [comment]
Should we move settings to the footer? It looks weird in the user profile area. (like a settings button)

**Comment by elinarbur** [comment]
To be fair it does look a bit awkward where it's at currently so that would definitely make it more accessible

**Comment by f1shy-dev** [comment]
Yeah, then clicking on your profile would be a popup like when you click on a user in a room, and not a full screen page.

**Comment by elinarbur** [comment]
Sounds good

**Comment by goldyydev** [comment]
@vishy-dev We're also going to need to implement it in the chat
![image](https://user-images.githubusercontent.com/55450464/111884519-cf584600-89ca-11eb-8c8b-2b6f7f871fb1.png)

**Comment by f1shy-dev** [comment]
I’ll include that too!
